doc/modules/model_evaluation.rst

```diff
@@ -2185,49 +2185,46 @@ of 0.0.
     -0.552
 
 
-|details-start|
-**D2 Brier score**
-|details-split|
+.. dropdown:: **D2 Brier score**
```

Comment:
```suggestion
.. dropdown:: D2 Brier score
```

Reply:
I don't think this is needed and maybe (?) it will fix the rendering of the first sentence:

<img width="1364" height="464" alt="image" src="https://github.com/user-attachments/assets/34323815-e1ba-442f-9b5c-6033e25f854a" />

The first sentence should not be in it's own special box (see the dropdown above, where the first sentence is not in a box).

Reply:
Note you should check the rendered docs: https://scikit-learn.org/dev/developers/contributing.html#generated-documentation-on-github-actions